README.md

```diff
@@ -13,3 +13,65 @@ might help with the notation for mathematical expressions.
 
 $T(n) \in O(f(n)) \iff \exists c, n_0: T(n) \leq c \cdot f(n) \forall n \geq n_0$
 
+
+# Proving that $O(\log_2 n)$ is the same as $O(\log_5 n)$ using the formal definition of Big O notation.
+
+## Formal Definition of Big O Notation
+
+For two functions $f(n)$ and $g(n)$,
+
+$$
+f(n) \in O(g(n)) \Leftrightarrow \exists c > 0, n_0 > 0 : 0 \leq f(n) \leq c \cdot g(n), \forall n \geq n_0
+$$
+
+Our goal is to show that $\log_2(n) \in O(\log_5(n))$ and vice versa.
+
+## Proof:
+
+We can utilize the change of base formula for logarithms:
+
+$$
+\log_a(b) = \frac{\log_c(b)}{\log_c(a)}
+$$
+
+for any positive $a, b, c$, where $a \neq 1$, $b \neq 1$, and $c \neq 1$.
+
+### Proving $\log_2(n) \in O(\log_5(n))$
```

Comment:
This is not sufficient -- you're only showing that $\log_2(n) \in O(\log_5(n))$. What about all the other expressions that are $\in O(\log_2(n))$?